manifest: sdk-nrfxlib: nrf_modem: update library (#19998)
nrfconnect/sdk-nrf: samples/dect/dect_phy/hello_dect/README.rst

   The availability of the channels and the exact regulations for using them vary in different countries.
   See section 5.4.2 of `ETSI TS 103 636-2`_ for the calculation.

.. _CONFIG_BAND_GROUP_INDEX:

> BAND_GROUP_INDEX

Reviewer: ```suggestion
CONFIG_BAND_GROUP_INDEX
```

   The band group index defines which RF frequencies the modem will operate on.
   This is configured before the modem is activated.

>    All nRF91 series devices with DECT NR+ PHY support has support for band index 0.

Reviewer: ```suggestion
   All nRF91 Series devices with DECT NR+ PHY support have support for band index 0.
```

>    DECT NR+ band 4 (band index 1) is supported by the nRF9151 LACA A0 SiP for R&D evaluation purposes.

Reviewer: ```suggestion
   DECT NR+ band 4 (band index 1) is supported by the nRF9151 LACA A0 SiP for research and development evaluation purposes.
```

   The nRF9151 LACA A0AB SiP will be qualified for DECT NR+ band 4 support.

>    The band group index 0 and 1 can be selected by enabling the :ref:`CONFIG_BAND_GROUP_INDEX_0 <CONFIG_BAND_GROUP_INDEX_0>` and :ref:`CONFIG_BAND_GROUP_INDEX_1 <CONFIG_BAND_GROUP_INDEX_1>` Kconfig options, respectively.

Reviewer: ```suggestion
   To select the band group index 0 and 1, enable the :ref:`CONFIG_BAND_GROUP_INDEX_0 <CONFIG_BAND_GROUP_INDEX_0>` and :ref:`CONFIG_BAND_GROUP_INDEX_1 <CONFIG_BAND_GROUP_INDEX_1>` Kconfig options, respectively.
```

Reviewer: We would also need to define CONFIG_BAND_GROUP_INDEX_1 and CONFIG_BAND_GROUP_INDEX_0 in the docs to be linkable like CONFIG_BAND_GROUP_INDEX.

Reviewer: will fix the doc error as well.

Reviewer 2: Updated, used `:kconfig:option:` for the band group index 0 and 1, as we don't need to link them.

Reviewer 2: I see, thank you for clarifying. Updated now.
